service.upnext/resources/lib/monitor.py

```diff
@@ -0,0 +1,56 @@
+import xbmc
+import resources.lib.utils as utils
+from resources.lib.player import Player
+
+class Monitor(xbmc.Monitor):
+
+    def __init__(self, *args):
+        self.logMsg("Starting UpNext Service", 0)
+        self.logMsg("========  START %s  ========" % utils.addon_name(), 0)
```

Comment:
most of those log lines are just spam. The kodi log will already include most of the info here